go.mod

```diff
@@ -119,6 +118,7 @@ require (
 	github.com/tklauser/go-sysconf v0.3.9 // indirect
 	github.com/tklauser/numcpus v0.3.0 // indirect
 	go.opencensus.io v0.23.0 // indirect
+	go.uber.org/atomic v1.9.0 // indirect
```

Comment:
Interesting which package imports it, could you check `go mod why go.uber.org/atomic` maybe?

Reply:
I also wondered about the same, and it turns out, jaeger is using it:
```
github.com/zalando/skipper/tracing/tracers/jaeger
github.com/uber/jaeger-client-go/config
github.com/uber/jaeger-client-go
```

Reply:
Now, interestingly, `jaeger` is being deprecated in favor of OpenTelemetry SDKs
https://www.jaegertracing.io/docs/1.39/client-libraries/#deprecating-jaeger-clients

Reply:
Will probably need to create a new ticket and proceed with deprecating jaeger.

Reply:
Er can't just move to OTel, because it's a breaking change so I expect that it won't happen in the next years.

Reply:
I see. Then we can leave this as is, I suppose.